code/AdobeEdge.brs

```diff
@@ -970,19 +965,35 @@ function _adb_buildEdgeRequestURL(configId as string, requestId as string, edgeD
 end function
 
 function _adb_EdgeRequestWorker(stateManager as object) as object
+    if stateManager = invalid
+        _adb_log_debug("stateManager is invalid")
+        return invalid
+    end if
     instance = {
         _queue: [],
         _stateManager: stateManager
         _queue_size_max: 100,
 
-        queue: function(requestId as string, xdmData as object, timestamp as integer, configId = invalid as dynamic, ecid = invalid as dynamic, edgeDomain = invalid as dynamic) as void
+        queue: function(requestId as string, xdmData as object, timestamp as integer) as void
+            if _adb_isEmptyOrInvalidString(requestId)
```

Comment:
we will never have empty or invalid request id since we internally generate it.

Reply:
Discussed offline, keep it for now.